lib/graphql/static_validation.rb

```diff
@@ -1,15 +1,38 @@
-module GraphQL::StaticValidation
-end
-
 require 'graphql/static_validation/message'
 require 'graphql/static_validation/arguments_validator'
 require 'graphql/static_validation/type_stack'
 require 'graphql/static_validation/validator'
 require 'graphql/static_validation/literal_validator'
 
-rules_glob = File.expand_path("../static_validation/rules/*.rb", __FILE__)
+rules_glob = File.expand_path('../static_validation/rules/*.rb', __FILE__)
```

Comment:
Yup! I prefer to use single quotes when double ones are not needed. But in this case I did it for consistency with the ones above :blush:.